.gitignore

```
@@ -14,4 +14,8 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 
-*.stamp
\ No newline at end of file
+*.stamp
+
+# Ignore all bazel-* symlinks. There is no full list since this can change
+# based on the name of the directory bazel is cloned into.
+/bazel-*
```

nit: add trailing newline
```suggestion
/bazel-*

```